### PR TITLE
Release 2.30.1

### PR DESCRIPTION
**Pull request by jpogran**

### CODE REVIEW

**Review by ansgarm — APPROVED** [review]

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,3 +1,23 @@
+## [2.31.0] (2024-04-22)
```
[the remark this answers is not in the document]
Comment (dbanck) [reply CHANGELOG.md:1]: We also need a link for this headline at the bottom of the file

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,3 +1,21 @@
...
```
Comment (dbanck) [inline CHANGELOG.md:1]:
```suggestion
## [2.30.1] (2024-04-22)
```

File: CHANGELOG.md [hunk 3]
```diff
@@ -964,7 +982,8 @@ For information on prior major and minor releases, see their changelogs:
 
 
 <!-- Links to tag comparisons -->
-[Unreleased]: https://github.com/hashicorp/vscode-terraform/compare/v2.30.0...main
+[Unreleased]: https://github.com/hashicorp/vscode-terraform/compare/v2.31.0...main
+[2.31.0]: https://github.com/hashicorp/vscode-terraform/compare/v2.30.0...v2.31.0
```
Comment (dbanck) [inline CHANGELOG.md:986]:
```suggestion
[Unreleased]: https://github.com/hashicorp/vscode-terraform/compare/v2.30.1...main
[2.30.1]: https://github.com/hashicorp/vscode-terraform/compare/v2.30.0...v2.30.1
```

**Review by dbanck — CHANGES_REQUESTED** [review]
We're cutting `2.30.1`, right? 😄

**Review by dbanck — APPROVED** [review]
:shipit:

**Pull request merged**